portality/dao.py

```diff
@@ -860,18 +860,36 @@ def iterall(cls, page_size=1000, limit=None, **kwargs):
         # TODO: Another candidate for swapping to iterate_pit (or rename iterate_pit to iterate when we're happy)
         return cls.iterate(MatchAllQuery().query(), page_size, limit, **kwargs)
 
+    @classmethod
+    def iterall_unstable(cls, page_size=1000, stripe_field="id", striped=False, prefix_generator=None, **kwargs):
+        def hex_prefixes(n=3):
+            """ Generate a list of hex prefixes of length n """
+            return [str(hex(i))[2:].zfill(3) for i in range(0, 16 ** 3)]
+
+        if striped:
+            prefixes = prefix_generator() if prefix_generator is not None else hex_prefixes()
+            for prefix in prefixes:
+                q = {
+                    "query": {
+                        "prefix": {stripe_field: prefix}
+                    }
+                }
+                for record in cls.iterate_unstable(q, page_size, **kwargs):
```

Comment:
inner and outer `limit` kwargs will be different - the striped `limit` is applied per stripe whereas it's usually an overall document limit across the iterate.